plugins/CoreHome/angularjs/http404check.js

```diff
@@ -15,6 +15,23 @@
         }
 
         return {
+            'request': function(config) {
+                if ('object' === typeof piwik.relativePluginWebDirs
+                    && config && config.url && config.url.indexOf('plugins/') === 0
+                    && config.url.indexOf('.html') > 0
+                    && config.url.indexOf('/angularjs/') > 0) {
+
+                    var urlParts = config.url.split('/');
+                    if (urlParts && urlParts.length > 2 && urlParts[1]) {
+                        var pluginName = urlParts[1];
+                        if (pluginName && pluginName in piwik.relativePluginWebDirs) {
```

Comment:
The `pluginName` check in the if is redundant here since `urlParts[1]` is checked above.

Reply:
Would it be safer to check `if (piwik.relativePluginWebDirs[pluginName])` instead of `pluginName in piwik.relativePluginWebDirs` in case the value is falsey?

Reply:
Added another check. I think we could remove `plugin in ...` but had sometimes problems with it in past not sure why.